examples/src/main/scala/org/apache/spark/examples/mllib/FPGrowthExample.scala

```diff
@@ -18,29 +18,58 @@
 package org.apache.spark.examples.mllib
 
 import org.apache.spark.mllib.fpm.FPGrowth
-import org.apache.spark.{SparkContext, SparkConf}
+import org.apache.spark.{SparkConf, SparkContext}
+import scopt.OptionParser
 
 /**
  * Example for mining frequent itemsets using FP-growth.
+ * Example usage: ./bin/run-example org.apache.spark.examples.mllib.FPGrowthExample
```

Comment:
Same here, remove `org.apache.spark.examples.`. If the options doesn't fit into one line, use `\` at the end of the first line and indent the second.